package.json

```diff
@@ -35,7 +35,7 @@
     "@types/deep-freeze": "^0.1.1",
     "@types/express": "^4.16.1",
     "@types/express-session": "^1.15.12",
-    "@types/lodash": "4.14.123",
+    "@types/lodash": "^4.14.133",
```

Comment:
Are you so confident about their stability and decided to use `^`?
I'm fine with it, but I'm still hesitant...

Reply:
Using a fixed version can cause problems downstream where you have different versions of the same types so having a floating version makes this easier for downstream, and since we use a declaration file we should be safe from a floating version on publish and it'll only bite us on rerunning publish with newer types.  Overall bluebird and lodash types are by far the most problematic but it's a really awkward trade off with floating versions and downstream - I wrote https://github.com/balena-io/pinejs/pull/296 primarily to make dealing with differing bluebird types much less of an issue for downstream.

Anyway, I'm not confident about either bluebird or lodash types, but the floating version has some benefits for downstream life